packages/arb-token-bridge-ui/src/hooks/useTransactions.ts

```diff
@@ -88,6 +88,7 @@ type TransactionBase = {
   assetType: AssetType
   tokenAddress?: string
   sender: string
+  destination?: string
```

Comment:
There's an inconsistency between naming here. You can see `destination` now and `destinationAddress` in other places. However, there are inconsistencies in our existing types (some of which come from sdk) which force such properties here.